crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -185,6 +187,18 @@ pub struct CheckpointStoreTables {
     pub(crate) watermarks: DBMap<CheckpointWatermark, (CheckpointSequenceNumber, CheckpointDigest)>,
 }
 
+fn full_checkpoint_content_table_default_config() -> DBOptions {
+    DBOptions {
+        options: default_db_options()
+            .optimize_for_large_values_no_scan(1 << 10)
```

Comment:
Let's not enable this option for now. `full_checkpoint_content` table prunes its entries pretty aggressively (5000 checkpoints). I'm a bit worried about blob gc being more costly than rocksdb compactions.